doc/source/dev/contributor/quickstart_ubuntu.rst

```diff
@@ -2,9 +2,9 @@
 
 .. _quickstart-ubuntu:
 
-=================================================
-Development Environment Quickstart Guide (Ubuntu)
-=================================================
+=======================================================
+Development environment quickstart guide (Ubuntu 16.04)
```

Comment:
Apparently section headings are supposed to be lower case? This is very inconsistent, but the other quickstarts were changed in gh-10823 so I guess this should follow suit.